src/box/raft.c

```diff
@@ -505,33 +505,40 @@ box_raft_try_promote(void)
 	};
```

Comment:
Nit: in the commit msg of the first commit: The hang ~~is~~ happened...
`is` not needed there

And let's also describe, why did we change the order of errors in commit msg